### ISSUE TITLE
question

### ISSUE THREAD

**Issue by hackerwei** [lukovkin/ufcnn-keras#9]
Hi Dmitry:
Does the UFCNN simulation show that CNN works better than LSTM RNN? I have read the paper, but I take a sceptical attitude. @lukovkin

**Comment by lukovkin** [comment]
Hi, @hackerwei!

We were thinking and experimenting a lot with this paper, and I came to the conclusion that though it's interesting and novel in some aspects, it's not pretty accurate and the final results are doubtful.

But our further experiments shown that CNN built in the some specific manner could show much better results than LSTM or GRU. On the other hand, the CNN block could be used in front of the LSTM block (though this setup hadn't delivered accuracy improvements in our case).

Also, WaveNet deliverables prove the fact that CNNs could be used for time series generation/prediction very efficiently.
Some aspects of the WaveNet are pretty similar to the proposed UFCNN architecture, but some are very different.
I found that WaveNet could be applied to the task, but suffers from the overfitting, we'll try to conduct additional experiments later.
